tools/downloader/common.py

```diff
@@ -321,8 +321,11 @@ def load_topologies(config):
 # requires the --print_all, --all, --name and --list arguments to be in `args`
 def load_topologies_from_args(parser, args):
     if args.print_all:
+        print_list = []
```

Comment:
I would prefer a more Pythonic way to populate this, e.g. `print_list = [top.name for top in load_topologies(args.config)]`.

Reply:
@IRDonch thanks for reviewing and sharing on Pythonic way. I had made the suggested changes and rebase.